src/js/_enqueues/lib/comment-reply.js

```diff
@@ -162,6 +163,7 @@ window.addComment = ( function( window ) {
 		var cancelLink = this;
 		var temporaryFormId  = config.temporaryFormId;
 		var temporaryElement = getElementById( temporaryFormId );
+		var headingText = temporaryElement.textContent;
```

Comment:
Thinking it's possible that themes would overwrite (or replace) the comments template? Perhaps old themes, or ... special themes? :)

As this is old vanilla js, may be good to make it a but bigger but always check of particular node exists before using it.

Then the above would be moved under `if ( ! temporaryElement || ! respondElement ) ...`.

Reply:
This line is below on line 168 of this file.